### PR TITLE
Check for floating point number and reformat it.

### PR DESCRIPTION
**Pull request by ami-day**
dcp-79
Fix floating point error issue in idf file. It causes the tests to fail. The pmid should not be a floating point number, it should be a string or integer.

### CODE REVIEW

File: build/lib/hca_to_scea/helpers/cell_lines.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import pandas as pd
```
Comment (jacobwindsor) [inline build/lib/hca_to_scea/helpers/cell_lines.py:1]: Ths build directory should not appear in any branch. Please remove and gitignore it

**Review by jacobwindsor — CHANGES_REQUESTED** [review]
I don't see the reason for this. Wouldn't it be better just to fix the input data than cater for an edge case of having float values for a PMID? A float value for a PMID is not valid anyway so we should not have that in our DB and if we do we should fix it. We shouldn't be catering for invalid data, we shouldn't allow invalid data

**Comment by idazucchi** [comment]
@amnonkhen to review

File: hca-to-scea/script_spreadsheets/SRP151260/E-HCAD-44.idf.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (amnonkhen) [inline hca-to-scea/script_spreadsheets/SRP151260/E-HCAD-44.idf.txt:1]: What is this `scripts_scpreadsheets` directory?
Which script uses it?
We already have a directory for the test inputs and the expected outputs.
The directory of files generated during the test run should not be added to git. If it is, delete it and add an entry to gitignore.

**Review by amnonkhen — CHANGES_REQUESTED** [review]

**Comment by ami-day** [comment]
Im closing this pull request as this has already been done and merged in a separate branch.

**Pull request closed without merging**